ex1_bod_css.css

```diff
@@ -0,0 +1,31 @@
+.div1 {
+	background-color: white;
+	height: 510px;
+	margin-left: 22%
+}
+
+.div2 {
+	background-color: #EFEEEC;
+	height: 510px;
+}
+
+.image {
+	width: 22%;
+	margin-right: 11%;
+}
+
+.foot
+{
+	height: 100%;
+	width: 100%;
+	background-color: #DFDFDF;
+	color: #828384;
+}
+
+.foot h2
+{
+	margin-top: 0;
+	margin-left: 68%;
```

Comment:
I tried those percentages till i get a better screen view. It's not a good way to do that, i noticed that today too